### DISCUSSION TITLE
Need a very basic C# example to anaylze arabic words

### DISCUSSION THREAD

**Discussion by MonsterMMORPG** [sergey-tihon/Stanford.NLP.NET#112] [Ideas]
I have created a new WPF project in visual studio 2017

Then from manage nuget packages > installed Stanford.NLP.CoreNLP

Then i have downloaded and extracted stanford-parser-full-2016-10-31

Then extracted stanford-parser-3.7.0-javadoc.jar, stanford-parser-3.7.0-models.jar and stanford-parser-3.7.0-sources.jar files into folders

Here inside of that folder

![inside folder](https://cloud.githubusercontent.com/assets/4149474/23775405/94fef3b0-0539-11e7-8901-8737eda2fb62.png)

Ok from here i need a very basic c# example for how to analyze arabic words?

**I would like to learn is word noun verb etc
Is word first person second person etc
Is word female or male or he she it etc
Is word plural or not or available other forms etc
Is word superlative or comparative etc**

Thank you very much

![need example](https://cloud.githubusercontent.com/assets/4149474/23775441/bb4e64b0-0539-11e7-9d19-0dc167ed4e62.png)

**Comment by sergey-tihon** [comment]
Start from this sample project https://github.com/sergey-tihon/Stanford.NLP.NET/tree/master/samples/Stanford.NLP.CoreNLP.CSharp and make sure that it works on you machine.

This discussion may be useful https://github.com/sergey-tihon/Stanford.NLP.NET/issues/18
